ComposeStarter/app/src/main/java/com/example/android/wearable/composestarter/presentation/ui/MainActivity.kt

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2021 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.example.android.wearable.composestarter.presentation.ui
+
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.wear.compose.material.ExperimentalWearMaterialApi
+import androidx.wear.compose.material.MaterialTheme
+import androidx.wear.compose.material.Text
+import com.example.android.wearable.composestarter.R
+import com.example.android.wearable.composestarter.presentation.theme.WearAppTheme
+
+/**
+ * Simple "Hello, World" app meant as a starting point for a new project using Compose for Wear OS.
+ *
+ * Displays only a centered [Text] composable, and the actual text varies based on the shape of the
+ * device (round vs. square/rectangular).
+ *
+ * If you plan to have multiple screens, use the Wear version of Compose Navigation. You can carry
+ * over your knowledge from mobile and it supports the swipe-to-dismiss gesture (Wear OS's
+ * back action). For more information, go here:
+ * https://developer.android.com/reference/kotlin/androidx/wear/compose/navigation/package-summary
+ */
+@ExperimentalWearMaterialApi
+class MainActivity : ComponentActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        setContent {
+            WearApp("Android")
+        }
+    }
+}
+
+@ExperimentalWearMaterialApi
```

Comment:
Is this actually needed? I don't see anything experimental in the sample.

Reply:
Removed, see note above.